### ISSUE TITLE
[Wayland] Ctrl+E, Ctrl+T, etc. work without entry list focus

### ISSUE THREAD

**Issue by nmschulte** [keepassxreboot/keepassxc#7211]
When unlocking and searching for an entry, using Ctrl + C works to copy to the clipboard the password of the selected (top, first) entry.  However, Ctrl + E, and others, only work to $action the selected (top, first) entry, when you first give the entry list focus; e.g. by pressing the "down" key (to change focus, to navigate the list/change list selection) after searching.

This is frustrating when I expect Ctrl + E to work the same way as Ctrl + C, but it does not, and instead nothing occurs at all.

I'd argue this issue is a bug and not a feature request.

**Comment by droidmonkey** [comment]
What version of keepassxc are you using

**Comment by nmschulte** [comment]
> What version of keepassxc are you using

`KeePassXC 2.6.6`
`keepassxc (2.6.6+dfsg.1-1) unstable; urgency=medium`

**Comment by droidmonkey** [comment]
I cannot replicate this on Windows on latest develop build. Please test a snapshot: https://snapshot.keepassxc.org

**Comment by nmschulte** [comment]
I see the same behavior with this snapshot: https://snapshot.keepassxc.org/latest/KeePassXC-2.7.0-snapshot-x86_64.AppImage


As well, probably best to consider this separately but be aware of it, there seems some inconsistency between...
- what "entry" is selected/focused, between the (filtered / "search results") list and the tree
- when the "entry" menu items are enabled
- when the "entry" menu items function
- the "entry" details displayed in the information area

It's possible to display a tree-item entry in the information area, with the focus on the search box, and the "entry" menu items are not enabled (except to create a new one, which seems most always enabled).  This is inconsistent with how these things behave when a list-item entry is displayed in the information area.  (Call "an entry displayed in the information area" as "in context"...?)

It's also possible to display a list-item entry in the information area when the list appears empty, due to a search with no results.

**Comment by droidmonkey** [comment]
What operating system and desktop env are you using. Also please provide screenshots of your above it is hard to follow the text.

**Comment by nmschulte** [comment]
I am running Sway (Wayland) on Debian (bookworm/+), with `QT_QPA_PLATFORM=`.  I am trying to find a working screenkeys solution so I can get you a recording of what I've wrote.

**Comment by droidmonkey** [comment]
Ahh so you are running Wayland, thus QWayland platform. There are a lot of bugs with focusing associated with that...

**Comment by nmschulte** [comment]
https://user-images.githubusercontent.com/8540239/145312287-acd42c27-589b-4aaf-afe2-fc7c90002278.mp4

**Comment by droidmonkey** [comment]
Cannot fix this, it is an upstream problem
